examples/resources/awscc_rds_db_parameter_group/awscc_rds_db_parameter_group_create_before_destroy.tf

```diff
@@ -0,0 +1,24 @@
+resource "awscc_rds_db_parameter_group" "this1" {
+  description = "rds sample db parameter group"
+  family      = "mysql5.7"
+
+  parameters = {
+    "character_set_server" = "utf8"
+    "character_set_client" = "utf8"
+  }
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "awscc_rds_db_instance" "this" {
```

Comment:
this resource is not required to complete the example, so I suggest to skip / remove this resource block

Reply:
Removed suggested resource block